docs/permission/overview.md

```diff
@@ -0,0 +1,41 @@
+---
+id: overview
+title: Permission Framework Overview
+description: A high level overview of the Backstage permission framework
+---
+
+[The previous section](../auth/index.md) covered the various _authentication_ methods of Backstage, but Backstage can also _authorize_ specific data, APIs, or interface actions - meaning that Backstage has the ability to enforce rules about what actions are allowed for a given user of a system.
+
+By default, Backstage endpoints are not protected and all actions are available to anyone. However, configuring which users can access which resources and actions is a common need for many organizations. The permission framework allows integrators to achieve this through the use of granular permissioning for those resources and actions.
+
+The permission framework was designed with a few key properties in mind:
+
+- Flexibility: the framework allows integrators to configure many different authorization methods. This could include implementations like role-based access control (RBAC), attribute-based access control (ABAC), bespoke logic expressed in code, or integrations with external authorization providers.
+
+- Usability: the permission framework allows integrators to focus on configuring what they care about (the permission policy) by providing all of its moving parts out of the box. It also allows plugin authors to integrate support for permissions in their plugins without having to make any changes in Backstage core.
+
+## How does it work?
```

Comment:
A plug for embedding the video demo somewhere on this page. https://www.youtube.com/watch?v=EQr9tFClgG0

Reply:
Yes! Thought of that right after I pushed it up but didn't get around to finding the link, thanks for doing that for me 😄

Reply:
On second thought, this probably belongs on the adopters documentation but it'll for sure be linked there!